dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -227,13 +227,13 @@ FEATURE_NOT_IMPLEMENTED.useraction=Avoid using the feature, or switch to a drive
 # Using 4000-4999 for Liberty-only messages
 # 
 
-DSRA4000.no.suitable.driver=DSRA4000E: A valid JDBC driver implementation class was not found for the jdbcDriver {0} using the library {1}.
-DSRA4000.no.suitable.driver.explanation=A valid JDBC driver implementation class was not found.
-DSRA4000.no.suitable.driver.useraction=Update the configuration so that either the library contains a path to a recognized JDBC driver or the implementation class is specified on the data source.
+DSRA4000.no.suitable.driver=DSRA4000E: No implementations of {0} are found for {1} with library {2}. Searched in: {3}. Searched under packages: {4}.
+DSRA4000.no.suitable.driver.explanation=Reasons why a data source or driver implementation might not be found include: Name or location of JDBC driver JAR files might be incorrect or inaccessible. JDBC driver might not properly identify its Driver class, or might be lacking one. Liberty's built-in logic to identify data source implementation classes does not apply to all drivers.
```

Comment:
These messages are the same for both scenarios. Is that correct? 

DSRA4001.no.suitable.driver.nested.explanation=A data source or driver implementation might not be found for several reasons. The name or location of JDBC driver JAR files might be incorrect or inaccessible. The JDBC driver might not properly identify its Driver class, or it might not have a Driver class. Or, Liberty's built-in logic to identify data source implementation classes does not apply to all drivers.

Reply:
Yes, the explanation/useraction are intentionally identical for the DSRA4000 and DSRA4001 messages.  The only difference is in whether we are able to supply a library identifier to the message.  This is possible when the library configuration element has an id, and isn't when the library configuration element is nested, lacking an id.